src/amo/pages/Guides/index.js

```diff
@@ -263,7 +275,7 @@ export const getContent = (slug: string, i18n: I18nType): GuideType | null => {
           and security.`,
         ),
         icon: 'stop-hand',
-        sections: getSections({ slug, i18n }),
+        sections: _sections || getSections({ slug, i18n }),
```

Comment:
note: I added this here because without this it was hard to test

Reply:
Why not naming this prop `_sections` then?